src/LaravelConsoleTaskServiceProvider.php

```diff
@@ -34,12 +34,14 @@ public function boot()
          *
          * @param  string $title
          * @param  callable|null $task
+         * @param  string $errorMessage
+         * @param  string $loadingText
          *
          * @return bool With the result of the task.
          */
         Command::macro(
             'task',
-            function (string $title, $task = null, $loadingText = 'loading...') {
+            function (string $title, $task = null, string $errorMessage = "failed", string $loadingText = 'loading...') {
```

Comment:
This seems breaking. Can you pass the new argument as optional parameter on the 4 place?

Reply:
@nunomaduro  Ive changed this to the 4th parameter with the default text being `failed`. Ive also adjusted the tests to pass in both a custom loading text and error text property where it was applicable.